Sources/MapboxMaps/MapView/MapView+Supportable.swift

```diff
@@ -8,13 +8,16 @@ extension MapView: OrnamentSupportableView {
     }
 
     internal func compassTapped() {
-        // Don't have access to CameraManager, so calling UIView.animate directly.
-        UIView.animate(withDuration: 0.3,
-                       delay: 0.0,
-                       options: [.curveEaseOut, .allowUserInteraction],
-                       animations: { [weak self] in
-                        self?.cameraView.bearing = 0.0
-        }, completion: nil)
+        var animator: BasicCameraAnimator?
+        animator = camera.makeAnimator(duration: 0.3, curve: .easeOut, animations: { (transition) in
+            transition.bearing.toValue = 0
+        })
+
+        animator?.addCompletion({ (_) in
+            animator = nil
+        })
```

Comment:
Not the only example of this in the PR — I'd love to see us using trailing closure syntax more consistently